tests/interfaces/test_batch_autograd.py

```diff
@@ -98,7 +106,7 @@ def cost(a, device):
                 qml.RX(a[1], wires=0)
                 qml.expval(qml.PauliZ(0))
 
-            return execute([tape], device, gradient_fn=param_shift, interface=None)[0]
+            return execute([tape], device, gradient_fn=param_shift, interface="None")[0]

```

Comment:
Why the change to a str?

Reply:
Because the recent PR change to PL means that `interface=None` now has semantic meaning that I have to follow 😆